packages/experiments/src/components/VerticalPersona/VerticalPersona.types.ts

```diff
@@ -1,25 +1,28 @@
 import { IStyle, IFontWeight } from '@uifabric/styling';
 import { IStatelessComponent, IStyleableComponentProps } from '../../Foundation';
-import { IPersonaCoinProps } from '../PersonaCoin/PersonaCoin.types';
+import { ITextSlot, IHTMLDivSlot, IPersonaCoinSlot } from '@uifabric/experiments/lib/utilities/factoryComponents.types';
 
 export type IVerticalPersonaComponent = IStatelessComponent<IVerticalPersonaProps, IVerticalPersonaStyles>;
 
+export interface IVerticalPersonaSlots {
+  root?: IHTMLDivSlot;
+  primaryText?: ITextSlot;
+  secondaryText?: ITextSlot;
+  coin?: IPersonaCoinSlot;
```

Comment:
As we discussed in chat, I think all slots should have representation in `IVerticalPersonaStyles` to allow access via theming.